system/ee/ExpressionEngine/View/_shared/thumb.php

```diff
@@ -58,7 +58,7 @@
 
                 <?php if ($i == 1) : ?>
                     <div class="file-thumbnail__wrapper">
-                        <a<?php foreach ($row['attrs'] as $key => $value) :?> <?=$key?>="<?=$value?>"<?php endforeach; ?> data-file-id="<?=$row['attrs']['file_id']?>">
```

Comment:
@Yulyaswan is there specific reason all extra attributes are being removed?

Reply:
@intoeetive  before these changes, the main wrapper for each element was `<a>` with all attributes, then I changed the wrapper to a `<div>` to fix the issue of `"Checkbox in thumbnail view opens an image when it shouldn't" `and put the `<a>` inside a block.
In this case, all attributes are duplicated, which can be a problem in the future. That's why I left all attributes for the main wrap and only the `href` for the link.